roles/ceph-grafana/tasks/configure_grafana.yml

```diff
@@ -72,13 +72,6 @@
     group: "grafana"
     recurse: true
 
-# TODO!!! Figure out how to distribute these !!!TODO
-- name: Store the dashboard files
-  copy:
-    src: "files/dashboards/"
-    dest: "{{ grafana.dashboards_path }}"
```

Comment:
We can probably also remove the `grafana.dashboard_path` variable in this commit.

Reply:
Yeah, this is all gone now